### ISSUE TITLE
[Discovery|Development] Claims Status: Determine Possible Claim Return Types from First Layer Back End

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#11801]
## Issue Description
This is part 1 of what will likely be a 4-5 part series.  This ticket is to understand how we get the very generic "we don't know the progress of your claim" error back.  This ticket is to discover the possible error types and then determine whether we can make some content updates to be more informative. This is very low hanging fruit fix and not the full "have a good thing to say about every claim/appeal status" type.

Here are examples of what this looks like to a veteran:

<details><summary>Screenshots</summary>

![20200727_151045](https://user-images.githubusercontent.com/136959/89463035-eeec7800-d733-11ea-8485-235ef1f01992.jpg)
![20200727_151025](https://user-images.githubusercontent.com/136959/89463049-f449c280-d733-11ea-9c08-f294f713de12.jpg)
![20200727_151001](https://user-images.githubusercontent.com/136959/89463050-f57aef80-d733-11ea-86de-720e0671e393.jpg)</details>

And here is what Robin is saying we have as possible types: 

`{
  pendingSoc: 'pending_soc',
  pendingForm9: 'pending_form9',
  pendingCertification: 'pending_certification',
  pendingCertificationSsoc: 'pending_certification_ssoc',
  remandSsoc: 'remand_ssoc',
  pendingHearingScheduling: 'pending_hearing_scheduling',
  scheduledHearing: 'scheduled_hearing',
  onDocket: 'on_docket',
  atVso: 'at_vso',
  decisionInProgress: 'decision_in_progress',
  bvaDevelopment: 'bva_development',
  stayed: 'stayed',
  remand: 'remand',
  merged: 'merged',
  bvaDecision: 'bva_decision',
  fieldGrant: 'field_grant',
  withdrawn: 'withdrawn',
  ftr: 'ftr',
  ramp: 'ramp',
  reconsideration: 'reconsideration',
  death: 'death',
  otherClose: 'other_close',
  evidentiaryPeriod: 'evidentiary_period',
  amaRemand: 'ama_remand',
  postBvaDtaDecision: 'post_bva_dta_decision',
  bvaDecisionEffectuation: 'bva_decision_effectuation',
  scReceived: 'sc_received',
  scDecision: 'sc_decision',
  scClosed: 'sc_closed',
  hlrReceived: 'hlr_received',
  hlrDtaError: 'hlr_dta_error',
  hlrDecision: 'hlr_decision',
  hlrClosed: 'hlr_closed',
  statutoryOptIn: 'statutory_opt_in',
}`

## Acceptance Criteria
- [ ] Work on new design and verbiage (info box?) on how to add details as to why a claim status may fall into this unknown category.

## How to configure this issue
- [x] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by Mottie** [comment]
I spoke with the EVSS team and discovered that these statuses are not from EVSS, but instead come from our API.

After searching & matching, I found that the status list we have on the front-end matches the API statuses. So where ever the API is obtaining these statuses must have additional values we don't have:

### Statuses found in two files:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/schemas/appeals.json#L92
https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/caseflow/schema/appeals.json#L114

- [x] pendingSoc: 'pending_soc'
- [x] pendingForm9: 'pending_form9'
- [x] pendingCertification: 'pending_certification'
- [x] pendingCertificationSsoc: 'pending_certification_ssoc'
- [x] remandSsoc: 'remand_ssoc'
- [x] pendingHearingScheduling: 'pending_hearing_scheduling'
- [x] scheduledHearing: 'scheduled_hearing'
- [x] onDocket: 'on_docket'
- [x] atVso: 'at_vso'
- [x] decisionInProgress: 'decision_in_progress'
- [x] bvaDevelopment: 'bva_development'
- [x] stayed: 'stayed'
- [x] remand: 'remand'
- [x] merged: 'merged'
- [x] bvaDecision: 'bva_decision'
- [x] fieldGrant: 'field_grant'
- [x] withdrawn: 'withdrawn'
- [x] ftr: 'ftr'
- [x] ramp: 'ramp'
- [x] reconsideration: 'reconsideration'
- [x] death: 'death'
- [x] otherClose: 'other_close'

### Statuses only found in one file

https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/caseflow/schema/appeals.json#L137

- [x] evidentiaryPeriod: 'evidentiary_period'
- [x] amaRemand: 'ama_remand'
- [x] postBvaDtaDecision: 'post_bva_dta_decision'
- [x] bvaDecisionEffectuation: 'bva_decision_effectuation'
- [x] scReceived: 'sc_received'
- [x] scDecision: 'sc_decision'
- [x] scClosed: 'sc_closed'
- [x] hlrReceived: 'hlr_received'
- [x] hlrDtaError: 'hlr_dta_error'
- [x] hlrDecision: 'hlr_decision'
- [x] hlrClosed: 'hlr_closed'
- [x] statutoryOptIn: 'statutory_opt_in'

**Comment by Mottie** [comment]
While I searched through caseflow's repository, it appears that all the above events were accounted for, and I did not discover any missing events. Luckily @edmangimelli spotted a Sentry entry - http://sentry.vfs.va.gov/vets-gov/website-production/issues/34946/?query=appeals-unknown-event - that shows the error is from a `remand_return` status.

`remand_return` is listed as a legacy event in [this file](https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/models/appeal_events.rb#L32) along with a list of other legacy events. I've started a discussion with caseflow in the `#appeals-api` channel, and will loop in @kevinstachura.
